src/components/Onboarding/index.js

```diff
@@ -209,6 +209,8 @@ const Container = styled(Box).attrs({
   right: 0;
   bottom: 0;
   z-index: 25;
+  user-select: text;
+  cursor: text;
```

Comment:
is there a reason to not use the `selectable` attribute here?

Reply:
Actually not really. I guess if we decide to not use Box and use something else in those wrappers usage of selectable would be a very small issue compare to not using Box at all😁
Will change it